packages/contract/src/call-test-contract/call-test-contract.test.ts

```diff
@@ -16,7 +16,7 @@ const setup = async (abi: ReadonlyArray<JsonFragment> | Interface = abiJSON) =>
 
   // Create wallet
   const wallet = Wallet.generate({ provider });
-  await seedWallet(wallet, [{ assetId: NativeAssetId, amount: 1 }]);
+  await seedWallet(wallet, [[1, NativeAssetId]]);
```

Comment:
This type of interface looks practical but same time not clear... I would suggest keeping an object even if you need to repeat the object keys. Or if we keep it this way maybe would be clear to be `[coinId, amount]`.
```
const coins = await sender.getCoinsToSpend([
  [amount.mul(2), assetIdA],
  [amount.mul(2), assetIdB],
]);
```

Reply:
I chose that order because:
- We already had `transfer` implemented in fuels-rs and it was AssetId-last. (`dest,amount,asset_id` and not `dest,asset_id,amount`)
- "Amount then unit"  feels natural:
  `2 stacks of Saronite Ore` -> `[2 * STACK_SIZE, SARONITE_ORE]`
  `100 dollars and 52 euros` -> `[[100, USDC], [52, EURS]]`

Reply:
I think makes sense to keep the consistency between the `rs and ts`.  Even if I think the array API is a bit strange... @AlicanC thanks